From b823500f3390339ac3b75f7b65b2ceca097b5f2e Mon Sep 17 00:00:00 2001
From: Eugene Syromiatnikov <esyr@redhat.com>
Date: Thu, 10 Aug 2023 18:49:54 +0200
Subject: [PATCH 06/10] releasenote.md: fix stepping for RPL-S

microcode-20221108, microcode-20230214, microcode-20230512,
and microcode-20230512-rev2 release notes (incorrectly) state RPL-S
(06-b7-01/32) stepping as S0, while microcode-20230808 release notes
state it as B0, and [1] confirms the correctness of the latter.

[1] "13th Generation Intel Core Processors.  Datasheet, Volume 1 of 2"
    Rev. 005, February 2023, section 15.0 "CPU And Device IDs"
    https://cdrdv2-public.intel.com/743844/743844-005.pdf

* releasenote.md (microcode-20221108, microcode-20230214,
microcode-20230512, microcode-20230512-rev2) <RPL-S>: Change
the stepping field value from "S0" to "B0".

Signed-off-by: Eugene Syromiatnikov <esyr@redhat.com>
---
 releasenote.md | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/releasenote.md b/releasenote.md
index a29889e..3e0e786 100644
--- a/releasenote.md
+++ b/releasenote.md
@@ -146,7 +146,7 @@
 | RKL-S          | B0       | 06-a7-01/02 | 00000057 | 00000058 | Core Gen11
 | RPL-H 6+8      | J0       | 06-ba-02/c0 | 0000410e | 00004112 | Core Gen13
 | RPL-P 6+8      | J0       | 06-ba-02/c0 | 0000410e | 00004112 | Core Gen13
-| RPL-S          | S0       | 06-b7-01/32 | 00000112 | 00000113 | Core Gen13
+| RPL-S          | B0       | 06-b7-01/32 | 00000112 | 00000113 | Core Gen13
 | RPL-U 2+8      | Q0       | 06-ba-03/c0 | 0000410e | 00004112 | Core Gen13
 | SKX-D          | H0       | 06-55-04/b7 | 02006e05 | 02006f05 | Xeon D-21xx
 | SKX-SP         | B1       | 06-55-03/97 | 01000161 | 01000171 | Xeon Scalable
@@ -212,7 +212,7 @@
 | RKL-S          | B0       | 06-a7-01/02 | 00000057 | 00000058 | Core Gen11
 | RPL-H 6+8      | J0       | 06-ba-02/c0 | 0000410e | 00004112 | Core Gen13
 | RPL-P 6+8      | J0       | 06-ba-02/c0 | 0000410e | 00004112 | Core Gen13
-| RPL-S          | S0       | 06-b7-01/32 | 00000112 | 00000113 | Core Gen13
+| RPL-S          | B0       | 06-b7-01/32 | 00000112 | 00000113 | Core Gen13
 | RPL-U 2+8      | Q0       | 06-ba-03/c0 | 0000410e | 00004112 | Core Gen13
 | SKX-D          | H0       | 06-55-04/b7 |          | 02006f05 | Xeon D-21xx
 | SKX-SP         | B1       | 06-55-03/97 | 01000161 | 01000171 | Xeon Scalable
@@ -274,7 +274,7 @@
 | JSL            | A0/A1    | 06-9c-00/01 | 24000023 | 24000024 | Pentium N6000/N6005, Celeron N4500/N4505/N5100/N5105
 | LKF            | B2/B3    | 06-8a-01/10 | 00000031 | 00000032 | Core w/Hybrid Technology
 | RKL-S          | B0       | 06-a7-01/02 | 00000056 | 00000057 | Core Gen11
-| RPL-S          | S0       | 06-b7-01/32 | 0000010e | 00000112 | Core Gen13
+| RPL-S          | B0       | 06-b7-01/32 | 0000010e | 00000112 | Core Gen13
 | SKX-SP         | B1       | 06-55-03/97 | 0100015e | 01000161 | Xeon Scalable
 
 
@@ -295,7 +295,7 @@
 | Processor      | Stepping | F-M-S/PI    | Old Ver  | New Ver  | Products
 |:---------------|:---------|:------------|:---------|:---------|:---------
 | ICL-D          | B0       | 06-6c-01/10 |          | 01000201 | Xeon D-17xx, D-27xx
-| RPL-S          | S0       | 06-b7-01/32 |          | 0000010e | Core Gen13
+| RPL-S          | B0       | 06-b7-01/32 |          | 0000010e | Core Gen13
 
 ### Updated Platforms
 
-- 
2.13.6

